_articles/mobile/files.md

```diff
@@ -60,12 +60,73 @@ The data APIs have additional APIs for workers and administrators (not discussed
 
 ## Participant Files API
 
-[Storing data files for a participant](/swagger-ui/index.html#/Participant%20Files) involves the following steps:
+The Participant File API allows a participant to upload files to S3 that are private to the participant themselves. When uploading a file, a request is made to reserve a `fileId` and the API returns a presigned URL for uploading the file's contents. Similarly, the file can be retrieved or deleted through the API. And a list of the user's own existing files can be retrieved.
 
-1. Caller creates a [ParticipantFile](/model-browser.html#ParticipantFile) record through the [create participant file API.](/swagger-ui/index.html#/Participant%20Files/createParticipantFile) Please note that the mime/type for the file is required, although Bridge will currently allow you to create a file record without it;
-1. The returned ParticipantFile that is returned will have an `uploadURL` field with a presigned URL that can be used to `PUT` the file to S3 (**note** that currently a file cannot be replaced, so this call will fail if the file already exists; this will be changed to allow overwrites of files in a future update);
-1. Client uploads the file to S3;
-1. The caller can retrieve a [list of created file identifiers](/swagger-ui/index.html#/Participant%20Files/getParticipantFiles);
-1. The caller can retrieve a file by calling the [retrieve file API](/swagger-ui/index.html#/Participant%20Files/getParticipantFile) which returns a 302 redirecting the client with a presigned URL to retrieve the file data from S3.
+### Get a list of files
 
-Participant files can also be [deleted.](/swagger-ui/index.html#/Participant%20Files/deleteParticipantFile) They are not currently available to researchers or study coordinators. 
+The [Get Participant Files](/swagger-ui/index.html#/Participant%20Files/getParticipantFiles) API retrieves a list of [Participant Files](/model-browser.html#ParticipantFile) that can be used to find currently stored files' IDs. This includes information about the file, such as MIME type and date uploaded.
+
+The list is in the form of a [ForwardCursorPagedResourceList](/model-browser.html#ForwardCursorPagedResourceList). The amount of results per call is set through the optional `pageSize` parameter. `pageSize` defaults to 50 and can be within 5-100.
+
+```
+.../v3/participants/self/files?pageSize=5
+```
+
+The response will include a list of [ParticipantFiles](/model-browser.html#ParticipantFile) in the `items` field. If there are more pages of results available to retrieve, the `hasNext` field will return `true` and an `offsetKey` will be included.
+
+ ```json
+    {
+    "items": [...],
+    "nextPageOffsetKey": "sample-offset-key",
+    "requestParams": {
+        "pageSize": 5,
+        "type": "RequestParams"
+    },
+    "pageSize": 5,
+    "offsetKey": "sample-offset-key",
+    "total": 5,
+    "hasNext": true,
+    "type": "ForwardCursorPagedResourceList"
```

Comment:
pageSize should be removed from the root JSON, as that field is deprecated. It's not in requestParams because it can be sent as a request parameter. total is also deprecated. hasNext and nextPageOffsetKey are still correct (along with requestParams).